### ISSUE TITLE
Have an easy way to update all the fields in a row

### ISSUE THREAD

**Issue by Ooxie** [python-bonobo/bonobo#285]
Bonobo version: 0.6.2 + #282
Python version: 3.5.2
Platform: Linux 4.15.0-30 - Ubuntu 16.04.5

Given a row with a lot of fields, is there an easy way to apply the same function to all of the fields at once?

This is an example with a node that takes a Bag as input and yields a Bag as output (important requirement). It works, but it is probably not optimal, although it rebuilds the Bag only once:
```python
@bonobo.config.use_raw_input
def apply_same_update_to_all_the_fields(row):
    yield type(row)._make([custom_update_or_builtin_function(e) for e in row])
```

There should probably be a more simple function in the API, or at least some documentation that shows the best practice, to achieve this.

**Comment by Paulholio** [comment]
I think something like this would be very useful if we could also define what the possible update types are. Some more useful examples might be to attempt to strip outer whitespace from all string data points in a row or to lowercase them all. Another example might be to read a file and convert all special characters (such as tabs or new lines) into something else.

**Comment by hartym** [comment]
Solved by #300, thanks to @KPilnacek
